### ISSUE TITLE
bcrypt not working

### ISSUE THREAD

**Issue by TheBlckbird** [nicbet/docker-phoenix#37]
I tried to scaffold the auth system, but bcrypt doesn’t work:
- `mix phx.gen.auth Accounts User users --live`
- `mix deps.get`
- `mix ecto.migrate`:
```
==> bcrypt_elixir
could not compile dependency :bcrypt_elixir, "mix compile" failed. Errors may have been logged above. You can recompile this dependency with "mix deps.compile bcrypt_elixir", update it with "mix deps.update bcrypt_elixir" or clean it with "mix deps.clean bcrypt_elixir"
==> phoenix_hoeraktion
** (Mix) "make" not found in the path. If you have set the MAKE environment variable, please make sure it is correct.
```

I am on macOS 14.0 and am using the latest version of Phoenix Framework.

**Comment by TheBlckbird** [comment]
The same happens with `argon2`

**Comment by TheBlckbird** [comment]
huh, there is no error thrown when I execute `mix ecto.migrate` from the execute tab in Docker Desktop
![image](https://github.com/nicbet/docker-phoenix/assets/66938161/c64b3771-bd61-4c33-beed-c54517c492c9)

**Comment by TheBlckbird** [comment]
huh, no error is thrown when executing `mix ecto.migrate` from Docker Desktop:
![image](https://github.com/nicbet/docker-phoenix/assets/66938161/a51cff0a-3074-4503-9618-cabe63eeb88e)

**Comment by TheBlckbird** [comment]
I've got it working with
```apk add build-base erlang-dev```

**Comment by TheBlckbird** [comment]
But Bcrypt will be still unavailable

**Comment by TheBlckbird** [comment]
I get the error `Module Bcrypt not found`

**Comment by nicbet** [comment]
@TheBlckbird sorry for the long wait. Work and family took a precedence over open-source the past weeks.

The Docker image does not include the `build-base` package that provides the `make` and `cc` commands to compile the `bcrypt_elixir` module. 

You can run `apk add build-base` and then retry `mix deps.compile` to resolve the issue (just tested working in a brand new container / app). 

I'll think for bit on whether we should include `build-base` in the image by default or not.

**Comment by TheBlckbird** [comment]
Thanks, I’ll try that out.
I think we should include `build-base` by default, because a lot of applications have some kind of encryption and authentication.

**Comment by nicbet** [comment]
Agreed, since the primary concern of the repo is to provide an easy out of the box dev experience, not a production runtime container, and many dependencies need to be compiled, we should have `build-base` baked in. 

I’ll add it to the next release.

**Comment by nicbet** [comment]
`build-base` package is now included by default starting from `1.7.5` release.

**Comment by TheBlckbird** [comment]
Thank you!
